test/e2e/e2e.go

```diff
@@ -478,10 +482,28 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 			}
 		}
 
-		err := podClient.Delete(context.Background(), podName, metav1.DeleteOptions{})
+		err = podClient.Delete(context.Background(), podName, metav1.DeleteOptions{})
 		framework.ExpectNoError(err, "should delete ovnkube-node pod")
 		framework.Logf("Deleted ovnkube-node %q", podName)
 
+		ginkgo.By("Verifyin that all nodes are schedulable and untainted")
+		err = wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			afterNodes, err := e2enode.GetReadySchedulableNodes(f.ClientSet)
```

Comment:
Technically we only need to get that one node on which the pod is getting deleted, but I like this. Its safe passage and doesn't really matter since the other nodes should be in a ready state anyways.